src/moscot/backends/ott/solver.py

```diff
@@ -200,27 +202,29 @@ def problem_kind(self) -> ProblemKind_t:  # noqa: D102
     @classmethod
     def _call_kwargs(cls) -> Tuple[Set[str], Set[str]]:
         geom_kwargs = {"epsilon", "relative_epsilon", "batch_size", "scale_cost", "cost_kwargs", "cost_matrix_rank"}
-        problem_kwargs = set(inspect.signature(LinearProblem).parameters.keys())
+        problem_kwargs = set(inspect.signature(linear_problem.LinearProblem).parameters.keys())
         problem_kwargs -= {"geom"}
         return geom_kwargs | problem_kwargs, {"epsilon"}
 
 
 class GWSolver(OTTJaxSolver):
-    """Solver solving quadratic optimal transport problem.
+    """Solver for the :term:`quadratic problem` :cite:`memoli:2011`.
 
-    The Gromov-Wasserstein (GW) problem involves two distribution in possibly two different spaces.
-    Points in the source distribution are matched to points in the target distribution by comparing the relative
-    location of the points within each distribution.
+    The :term:`Gromov-Wasserstein (GW) <Gromov-Wasserstein>` problem involves two distribution in
+    possibly two different spaces. Points in the source distribution are matched to points in the target distribution
+    by comparing the relative location of the points within each distribution.
```

Comment:
```suggestion
    by comparing the location of the points with respect to their distributions.
```

Reply:
No. this is mathematically wrong. Happy to keep it as it is.